src/Components/Trainers/Form/index.jsx

```diff
@@ -15,86 +19,68 @@ const Form = () => {
   const [shouldRedirect, setShouldRedirect] = useState(false);
   const dispatch = useDispatch();
   const trainers = useSelector((state) => state.trainers.data);
```

Comment:
I think you could avoid getting all the trainers for this form, you could use a getById instead